src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -976,6 +976,9 @@
   "TestAllIndexers": "Test All Indexers",
   "TestAllLists": "Test All Lists",
   "TheLogLevelDefault": "The log level defaults to 'Info' and can be changed in",
+  "Theme": "Theme",
+  "ThemeHelpText": "Themes maintained by Theme.Park!",
```

Comment:
Don't put `Theme.Park!` in the translation, it allows people to sub it out. `{0}` here and hardcode it.

Reply:
Done